mtest/libmscore/spanners/updateReference

```diff
@@ -1,15 +1,16 @@
 #!/bin/bash
 
-cp ../../../build.debug/mtest/libmscore/spanners/glissando-cloning02.mscx    glissando-cloning02-ref.mscx
-cp ../../../build.debug/mtest/libmscore/spanners/glissando01.mscx            glissando01-ref.mscx
-cp ../../../build.debug/mtest/libmscore/spanners/glissando-cloning01.mscx    glissando-cloning01-ref.mscx
-cp ../../../build.debug/mtest/libmscore/spanners/glissando-cloning02.mscx    glissando-cloning02-ref.mscx
-cp ../../../build.debug/mtest/libmscore/spanners/glissando-cloning03.mscx    glissando-cloning03-ref.mscx
-cp ../../../build.debug/mtest/libmscore/spanners/glissando-cloning04.mscx    glissando-cloning04-ref.mscx
-cp ../../../build.debug/mtest/libmscore/spanners/glissando-cloning05.mscx    glissando-cloning05-ref.mscx
+cp ../../../build.debug/mtest/libmscore/spanners/glissando-cloning02.mscx     glissando-cloning02-ref.mscx
```

Comment:
...and again, formatting changes.  Let me know if I can let this slide, of if you want me to revert the formatting.